build/scripts/check-for-loc-changes.ps1

```diff
@@ -0,0 +1,62 @@
+<#
+.SYNOPSIS
+
+Checks if a merge will result in changes to .xlf files, and returns a failure
+code if true.
+
+.DESCRIPTION
+
+The point of this script is to block changes to localized resources after we've
+entered a Loc freeze. It's meant to be run in a CI system prior to merging a PR.
+
+.PARAMETER base
+
+Generally the branch a change will be merged into, but this could also be a
+commit or tag.
+
+.PARAMETER head
+
+The commit that will be merged into $base. Again, this can be a branch, tag, or
+commit.
+#>
+
+[CmdletBinding(PositionalBinding=$false)]
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$base,
+
+    [Parameter(Mandatory=$true)]
+    [string]$head
+)
+
+Set-StrictMode -version 2.0
+$ErrorActionPreference="Stop"
+
+try {
+    . (Join-Path $PSScriptRoot "build-utils.ps1")
+    Push-Location $repoDir
+
+    # Find the merge base, then the list of files that changed since then.
+    $mergeBase = & git merge-base $base $head
+    $changedFiles = @(& git diff --name-only $mergeBase $head)
```

Comment:
You should be able to stick `-- *.xlf` at the end of the git diff, so then git will filter to just edits to .xlf files.